packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, IStyle, ITextFieldStyles, mergeStyles, TextField, useTheme } from '@fluentui/react';
+import React, { useState } from 'react';
+import { useLocale } from '../../localization';
+import { containerStyles, digitStyles, rowStyles, subStyles, textFieldStyles } from './Dialpad.styles';
+
+/**
+ * Strings of {@link Dialpad} that can be overridden.
+ *
+ * @public
+ */
+export interface DialpadStrings {
+  errorText: string;
+  defaultText: string;
+}
+
+/**
+ * Props for {@link Dialpad} component.
+ *
+ * @public
+ */
+export interface DialpadProps {
```

Comment:
can be done in a followup PR when this component is used, but this will need to expose a onChange property similar to textfield's onChange event so the developer using this component can use the dialpad output